### DISCUSSION TITLE
Rigth ToolBox not stable on scrolling. version 3.2.1

### DISCUSSION THREAD

**Discussion by zoutigo** [react-page/react-page#1102] [Q&A] [answered]
Hello.

The box on the rigth side (reziseBlock, .....).
How to make it stable on scrolling ?
It is quite difficult to get the correct scrolling position where it appear.

I don't know how to solve this issue , even if i guess it something regarding scroll and absolute position.

How to manage this ?

**Comment by macrozone** [comment, marked as answer]
@zoutigo can you show a video of what you mean?

looks stable to me:

https://user-images.githubusercontent.com/1972353/140895903-bfbec8c3-babb-4b0f-a66f-d0bd4c38cd1a.mov

**Comment by zoutigo** [comment]
hello.
I'm unable to make a short video at the moment.
This afternoon, i will.
But, from what i see , when the content is only one or 2 lines, it's difficult to get that box fixed.

**Reply by macrozone** [reply]
@zoutigo this is to allow for multiple editors on the same page. Each editor has their controls on the right side and if you scroll, they are sticky if you have not reached the top or bottom of the editor.
